docs/architecture.md

```diff
@@ -11,6 +11,52 @@ In pouch's roadmap, we set ecosystem embracing as a big target. To upper orchest
 
 ![Ecosystem Architecture](static_files/pouch_ecosystem_architecture.png)
 
+The ecosystem architecture may be a little bit complicated at first glance. Take it easy. We can get a thorough understanding of it from the following three dimensions.
+
+### Runtime Layer
+
+Runtime layer is located on the right-top of the architure picture. This dimension mainly focus on OCI-compatible runtimes supported in Pouch. These runtimes unifies specifications for standard on operating system process and application containers. Currently, Pouch will support four kinds of OCI-compatible runtimes:
+
+* runC
+* runlxc
+* runV
+* clear containers
+
+With runC, Pouch creates common containers like other container engine does, like docker. With runlxc, Pouch creates containers based on LXC. runlxc helps a lot when users need to run containers on a wide variaty of Linux kernels with the ability to be compatible with kernel 2.6.32+. Hypervisor-based containers has many scenarios as well. Pouch will support it with runV and clear container. 
+
+All these specific runtimes are supported under containerd. Containerd takes over all detailed container management, including creation, start, stop, deletion and so on.
+
+### Orchestration Layer
+
+Pouch is always active on supporting Kubernetes since the first day when it is designed. We illustrate this part on the top half of the archtecture picture. First, Pouch will integrate cri-containerd inside of itself, so Kubernetes will easily dominate Pouch to manage Pod. The workflow will pass cri-containerd, containerd client, containerd, runC/runV and pod. When configuring network of Pod, cri-containerd will take advantanges of network plugins which implement CNI interface.
```

Comment:
since the first day when it is -> remove "when"
inside of itself -> how about just "inside"
will easily dominate -> could easily ...